### PR TITLE
add sway support

### LINKED ISSUE #2: Please add support for sway
Here's an example on how it works in bash script: https://gitlab.com/msrd0/dotfiles/-/blob/master/HOME/.config/sway/catbg.sh#L13

### PR DESCRIPTION
**Pull request by LuckyTurtleDev**
fix #2

### CODE REVIEW

**Comment by LuckyTurtleDev** [comment]
@msrd0 please test

File: README.md [hunk 1]
```diff
@@ -62,11 +62,12 @@ WallpaperBuilder::new()?.set_wallpapers(|screen| {
...
 
 
- [__cargo_doc2readme_dependencies_info]: ggGkYW0BYXSEGyDwipHVMb5RGxgd3zutc1TvG3ARKV4UcQ1NGyM1aXabIPYbYXKEG_rTCvlocg5FG4NsVxJZCFQRG0o9IwnJzjjYGz_ns3T7pgtyYWSBg29tb3JlLXdhbGxwYXBlcnNlMC4zLjBvbW9yZV93YWxscGFwZXJz
+
+ [__cargo_doc2readme_dependencies_info]: ggGkYW0BYXSEG_W_Gn_kaocAGwCcVPfenh7eGy6gYLEwyIe4G6-xw_FwcbpjYXKEG7WNg5vbQg0-G2Kkqe08eywGG2qR1qNCmVseG02ad2s_-Wv5YWSBg29tb3JlLXdhbGxwYXBlcnNlMC4zLjBvbW9yZV93YWxscGFwZXJz
  [__link0]: https://crates.io/crates/wallpaper
  [__link1]: https://crates.io/crates/wall
  [__link10]: https://docs.rs/more-wallpapers/0.3.0/more_wallpapers/?search=set_wallpapers_from_vec
- [__link11]: https://docs.rs/more-wallpapers/0.3.0/more_wallpapers/?search=set_random_wallpapers_from_vec
+ [__link11]: `set_random_wallpapers_from_vec()`
```
Comment (msrd0) [inline README.md]: this change looks broken

Comment (LuckyTurtleDev) [reply README.md]: Yes, but I do not why. I have executed `doc2readme` and now the link is dead. I see no change the can cause this issue in this pr.

Comment (LuckyTurtleDev) [reply README.md]: I have forgot to add `--all-features`. The ci is also missing this. :thinking:

Comment (LuckyTurtleDev) [reply README.md]: But it locks like the `doc2readme --check` does not fail if the link is wrong.

Comment (LuckyTurtleDev) [reply README.md]: fixed

**Pull request merged**